samples/index_tuning_sample/README.md

```diff
@@ -0,0 +1,272 @@
+If you already have vector embeddings ready in your database, you can skip this step.
+Otherwise, create embeddings from the sample dataset `wine_reviews_dataset.csv` by running this command in your sample code directory:
+
+```bash
+python3 create_vector_embeddings.py
+```
+Now your database is populated with 100k vector embeddings.
+
+## Step 5 : Create Database Vector Store
+we create a database engine from the information you input previously, and create a vector store from that engine:
+
+```python
+from langchain_google_alloydb_pg import (
+    AlloyDBEngine,
+    AlloyDBVectorStore,
+)
+
+async def get_vector_store():
+    engine = await AlloyDBEngine.afrom_instance(
+        project_id=PROJECT_ID,
+        region=REGION,
+        cluster=CLUSTER_NAME,
+        instance=INSTANCE_NAME,
+        database=DATABASE_NAME,
+        user=USER,
+        password=PASSWORD,
+    )
+
+    vector_store = await AlloyDBVectorStore.create(
+        engine=engine,
+        distance_strategy=DISTANCE_STRATEGY,
+        table_name=vector_table_name,
+        embedding_service=embedding,
+    )
+    return vector_store
+```
+
+## Step 5:  Learn About Index Benchmarking Metrics
+It is time to measure the performance of this index on your dataset. Recall and latency are key metrics for assessing the performance of vector similarity search indexing. 
+
+### What is latency and how to measure it?
+Latency refers to the time it takes for the system to complete a search query and return results. Lower latency means faster search results and is crucial for user experience.
+
+To measure the query latency of an index, we measure how long it takes to perform a similarity search query against the index. After creating and applying the index, we put a timer around the similarity search statement, with the time difference being the query latency. 
+
+### What is recall and how to measure it? 
+In the context of ANN indexing (If you don’t know what ANN and KNN are, refer to this guide), index recall is determined by the proportion of accurate results to the total results returned. Accurate results are identified by the overlap between the current index's results and those obtained through brute force iteration (KNN). By dividing the count of accurate results by the total results returned, we obtain the recall of the index, indicating the index's search accuracy.
+
+## Step 6:  Index Benchmarking
+Run this command to get an index performance report printed out on your terminal:
+
+```bash
+python3 index_search.py
+```
+
+The sample code tested the recall and latency of both HNSW and IVFFlat on the current dataset. Let us understand how each measurement is implemented, so you could customize your own benchmarking script.
+
+We usually calculate the average latency of multiple queries to have a better understanding of the system's performance under different conditions and to reduce the impact of outliers and variability in individual measurements.
```

Comment:
"We usually" is vague language and I would avoid it if possible. I would remove the "usually" or replace it with "We will calculate..."